packages/block-library/src/block/edit.js

```diff
@@ -71,7 +72,6 @@ export default function ReusableBlockEdit( {
 	attributes: { ref },
 	__unstableParentLayout: parentLayout,
 } ) {
-	const { useLayoutClasses } = unlock( blockEditorPrivateApis );
```

Comment:
Unlocking at render time was actually quite a weird pattern 😅 thanks for catching it!